src/wp-login.php

```diff
@@ -1276,6 +1289,9 @@ function wp_login_viewport_meta() {
 
 	case 'login':
 	default:
+		?>
```

Comment:
Oops. These are too early now, even before the `doctype`:
```
		<h1 class="screen-reader-text">Log in</h1>
		<!DOCTYPE html>
	<html lang="en-US">
```

Reply:
Sorry; I should have moved this to draft. I just wanted to commit before doing dinner.